### ISSUE TITLE
[Usage]: how to use llm to get model forward result?

### ISSUE THREAD

**Issue by cxmeng** [vllm-project/vllm#19304]
### Your current environment

```text
The output of `python collect_env.py`
```

### How would you like to use vllm

I want to run inference of a [specific model](put link here). I don't know how to integrate it with vllm.
I want to test deepseekmtp scorer model and proposer model forward result,like this;
self.llm = LLM(
                    model=config.modelPath,
                    gpu_memory_utilization=0.85,
                    enforce_eager=True,
                    trust_remote_code=True,
                    tensor_parallel_size=1,
                    speculative_config={
                        "num_speculative_tokens": 1 # # add MTP config
                    }
                )
        self.pipe = self.llm.llm_engine.model_executor.driver_worker.proposer_worker.model_runner.model
        self.pipe1 = self.llm.llm_engine.model_executor.driver_worker.scorer_worker.model_runner.model

            device = next(self.pipe1.parameters()).device
            input_ids = self.tokenizer(prompt, return_tensors="pt").input_ids.to(device)
            position_ids = torch.arange(0, input_ids.shape[-1], dtype=torch.long, device=device)
            position_ids = position_ids.unsqueeze(0).expand_as(input_ids)
            self.pipe1.forward(input_ids, position_ids)

but it dosen't work, becasuse of forward_context = get_forward_context()??
i have give input_ids and position_ids  but i don't know how to get the hidden_states and logits

waiting for help, thanks

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by LugerW-A** [comment]
hi did you solve it?
